miniwin/miniwin/src/include/miniwin_d3drmmesh_p.h

```diff
@@ -17,18 +37,18 @@ struct Direct3DRMMeshImpl : public Direct3DRMObjectBase<IDirect3DRMMesh> {
 	) override;
 	HRESULT SetGroupColor(int groupIndex, D3DCOLOR color) override;
 	HRESULT SetGroupColorRGB(int groupIndex, float r, float g, float b) override;
+	HRESULT GetGroupColor(D3DRMGROUPINDEX index) override;
```

Comment:
This returns a color.
```suggestion
	D3DCOLOR GetGroupColor(D3DRMGROUPINDEX index) override;
```